x-pack/elastic-agent/pkg/agent/transpiler/ast.go

```diff
@@ -193,14 +216,14 @@ func NewList(nodes []Node) *List {
 
 func (l *List) String() string {
 	var sb strings.Builder
+	sb.WriteString("[")
```

Comment:
Was this a "bug" before?

Reply:
Yes it was. We only `ast.String()` for testing and development, so it doesn't have an effect on operations or even log output.